docs/user-guide/leverage-cli/private-repositories.md

```diff
@@ -31,3 +31,16 @@ $ eval "$(ssh-agent -s)"
 $ ssh-add ~/.ssh/<private_ssh_key_file>
 ```
 (replace `private_ssh_key_file` with the desired file, the process can request the passphrase if it was set on key creation step)
+
+## Using the SSH config file to specify the key that must be used for a given host
+
+The ssh-agent socket is not always available in all the OS, like in Mac. So now our `leverage terraform init` command copy the ssh config file (and the whole .ssh directory) into the container volume, which mean any custom configuration you got there,
+will be used. You can read more on the [ssh official documentation](https://www.ssh.com/academy/ssh/config).
```

Comment:
Just a small suggestion for change:

"So now our leverage terraform init command `copies` the ssh config file (and the entire .ssh directory) into the container volume, which `means` any custom configuration `you have there` will be used."

1. The verb "copy" should be in the third person singular form, which is "copies" in this case. 
2. Additionally, "mean" should be "means" to agree with the singular subject "which." 
3. Lastly, the phrase "you got there" should be changed to "you have there" for clarity and correct verb tense.